src/BlazorAdmin/Pages/CatalogItemPage/Edit.razor

```diff
@@ -85,7 +85,7 @@
                                         </div>
                                     </div>
 
-                                    <div class="form-group">
+                                    @*<div class="form-group">
```

Comment:
Same question and statement as a above.

Reply:
Will remove the commented/unnecessary code snippet.